dali/pipeline/operators/decoder/nvjpeg_decoder_cpu.cc

```diff
@@ -23,6 +23,7 @@ DALI_SCHEMA(nvJPEGDecoderCPUStage)
 It is automatically inserted during the pipeline creation.)code")
   .NumInput(1)
   .NumOutput(3)
+  .InternalOp()
```

Comment:
need to add `InternalOp` to MakeContiguous` too

Reply:
Added it in the beggining

Reply:
It's here https://github.com/NVIDIA/DALI/pull/652/files#diff-cc39b9b740a081220f80a3f5278844c2R25

Reply:
I don't see it in the PR

Reply:
Oh, I see it now, sorry :)